src/common/include/common.h

```diff
@@ -15,6 +15,8 @@ using LayerId          = size_t;
 using HeatmapId        = size_t;
 using RepresentationId = size_t;
 
+using Floats = std::vector<float>;
```

Comment:
Since you implemented new esuing replace the rest of default vectors of floats in code please